[2023-02-23 11:57:17,004] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_market_overview.push_market_overview scheduled__2023-01-30T17:00:00+00:00 [queued]>
[2023-02-23 11:57:17,032] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_market_overview.push_market_overview scheduled__2023-01-30T17:00:00+00:00 [queued]>
[2023-02-23 11:57:17,033] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-23 11:57:17,033] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-23 11:57:17,033] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-23 11:57:17,073] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): push_market_overview> on 2023-01-30 17:00:00+00:00
[2023-02-23 11:57:17,085] {standard_task_runner.py:52} INFO - Started process 716 to run task
[2023-02-23 11:57:17,091] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'get_market_overview', 'push_market_overview', 'scheduled__2023-01-30T17:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/get_market_over_view_dag.py', '--cfg-path', '/tmp/tmp7c9kd58p', '--error-file', '/tmp/tmpewl_p3mf']
[2023-02-23 11:57:17,093] {standard_task_runner.py:80} INFO - Job 24: Subtask push_market_overview
[2023-02-23 11:57:17,231] {task_command.py:369} INFO - Running <TaskInstance: get_market_overview.push_market_overview scheduled__2023-01-30T17:00:00+00:00 [running]> on host 003d565768d2
[2023-02-23 11:57:17,481] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_market_overview
AIRFLOW_CTX_TASK_ID=push_market_overview
AIRFLOW_CTX_EXECUTION_DATE=2023-01-30T17:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-30T17:00:00+00:00
[2023-02-23 11:57:20,305] {logging_mixin.py:115} INFO - (1062, "Duplicate entry '2023-02-23 00:00:00-KOSPI' for key 'market_overview.PRIMARY'")
[2023-02-23 11:57:26,471] {logging_mixin.py:115} INFO - (1213, 'Deadlock found when trying to get lock; try restarting transaction')
[2023-02-23 11:57:28,080] {logging_mixin.py:115} INFO - DONE!!!
[2023-02-23 11:57:28,081] {python.py:173} INFO - Done. Returned value was: None
[2023-02-23 11:57:28,109] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=get_market_overview, task_id=push_market_overview, execution_date=20230130T170000, start_date=20230223T115717, end_date=20230223T115728
[2023-02-23 11:57:28,160] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-23 11:57:28,209] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
